### ISSUE TITLE
Document Blacklight's goal towards WCAG 2.0 compliance

### ISSUE THREAD

**Issue by cbeer** [projectblacklight/blacklight#2086]
(From a discussion out of LDCX; ticket to come later)

**Comment by mejackreed** [comment]
Should we do this in a new Wiki page?
